spk/maraschino-custom/src/installer.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+# Package
+PACKAGE="maraschino-custom"
+DNAME="maraschino-custom"
```

Comment:
To keep in line with the other Custom packages, change this to `Maraschino Custom`